Purchases/Public/RCPurchases.h

```diff
@@ -152,11 +152,18 @@ NS_SWIFT_NAME(Purchases)
 
 /**
  @return A singleton `RCPurchases` object. Call this after a configure method to access the singleton.
+ @note: If the SDK has not been configured, calls to sharedPurchases will raise an exception. Make sure to configure the SDK before making calls to sharedPurchases.
  */
 @property (class, nonatomic, readonly) RCPurchases *sharedPurchases;
 
 #pragma mark Configuration
 
+/**
+ @note True if the SDK has been configured, false otherwise. This property should only be used in special circumstances. If the shared instance has not been configured,
+ calls made to it will raise an exception.
+ */
+@property (class, nonatomic, readonly) BOOL isConfigured;
```

Comment:
This doesn't exist in Android, and I'm not 100% sure it's needed. 

However, we are changing the behavior of the SDK, so maybe it's helpful for some folks if they're migrating behaviors. 

Note that the previous behavior is the no-op, which no one should really have been relying on.

Reply:
sounds good